apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -48,6 +48,12 @@
             if (errorMessage.equalsIgnoreCase("authentication.fail.message")) {
                 errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry.code.invalid");
             }
+            if (errorMessage.equalsIgnoreCase("token.expired")) {
+            	errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.code.expired.resend");
```

Comment:
when the config `disableOTPResendOnFailure=true`, the OTP will be sent to the user's email automatically. Hence in this case we can have a different error message to direct the user to get the OTP from the inbox. However, here in default case, why can't we re-use the existing  error message https://github.com/wso2/identity-apps/blob/bfa93d355eb171c8759e81563b24d7b281bfe676/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties#L194?

Do we really need to introduce new error message for this case too?

Reply:
In correcting this, when the config is `disableOTPResendOnFailure=false` only, the OTP will be sent to the user's mail automatically. 
`Do we really need to introduce new error message for this case too?` - The error message that has been used whenever the email has not been sent automatically is also an existing one [1] which is providing more clear error message which specifically says that the code entered is expired. `error.code.expired.resend=The code entered is expired. Click Resend Code to continue.`

[1] - https://github.com/wso2/identity-apps/blob/bfa93d355eb171c8759e81563b24d7b281bfe676/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties#L316

Reply:
As per the discussion, it has been planned to give two different error messages for expired otp flows 
1) Email sent automatically
2) Email not set automatically